chapter2/tests/t20/terraform/base/variables.tf

```diff
@@ -0,0 +1,85 @@
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ db vars                                                                             │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+ variable "rds_db_user" {
+  description = "db rds user"
+  type        = string
+  default     = "user_db"
+}
+
+ variable "rds_db_name" {
+  description = "rds db name"
+  type        = string
+  default     = "wordpress_db"
+}
+
+variable "efs_name" {
+  description = "Name of the EFS file system"
+  type        = string
+  default     = "wordpress-efs"
+}
+
+variable "availability_zones" {
+  description = "Availability Zones"
+  type        = list(string)
+}
+
+variable "vpc_tags" {
+  description = "VPC tags to place cache host into"
+  type        = map(string)
+  default = {
+    Name = "default"
+  }
+}
+
+variable "site_url" {
+  description = "site url"
+  type        = string
+  default     = "tf-maxim-omelchenko.saritasa-camps.link"
+}
+
+variable "subdomain_url" {
+  description = "subdomain url"
+  type        = string
+  default     = "tf-maxim-omelchenko"
+}
+
+
+variable "efs_id" {
```

Comment:
all your vars should be grouped by component and have the same prefix.

Reply:
you have efs_name above.

Reply:
> all your vars should be grouped by component and have the same prefix.

@dmitry-mightydevops 
Done